skema/metal/model_linker/README.md

```diff
@@ -0,0 +1,49 @@
+# SKEMA Model Linking Utility
+
+Link ASKEM models, such as `AMR` and `GrometFN` with text extractions in their [canonical form](https://github.com/ml4ai/ASKEM-TA1-DataModel).
+Currently, we support linking only to `AMR` representations, such as _PetriNets_ and _RegNets_.
+Soon, we will support linking to Gromet function networks.
+
+## Installation
+
+Clone this repository, and run:
+```shell
+pip install .
```

Comment:
Does this work if installed separately from the top-level pyproject.toml-based build?

Reply:
No. It was from before integrating it to the uber python package.
This goes away